src/providers/postgres/qgspostgresdataitems.cpp

```diff
@@ -324,6 +324,13 @@ QList<QAction *> QgsPGLayerItem::actions( QWidget *parent )
     lst.append( actionTruncateLayer );
   }
 
+  if ( mLayerProperty.isMaterializedView )
+  {
+    QAction *actionRefreshMaterializedView = new QAction( tr( "Refresh Materialized View %1" ).arg( typeName ), parent );
```

Comment:
Won't this be "refresh Materialised view view"?

Reply:
Oups sorry. Quick copy/paste